internal/commands/repo/rm.go

```diff
@@ -47,7 +47,7 @@ type rmOptions struct {
 func newRmCmd(streams command.Streams, hubClient *hub.Client, parent string) *cobra.Command {
 	var opts rmOptions
 	cmd := &cobra.Command{
-		Use:                   rmName + " [OPTIONS] REPOSITORY",
+		Use:                   rmName + " [OPTIONS] USERNAME(OR)ORGANIZATION NAME/REPOSITORY",
```

Comment:
question: I wonder 🤔 if we should just rephrase to:
```
NAMESPACE/REPOSITORY
```
WDYT ?

Reply:
tks again for the guidance, it's great to have your support!